parquet-variant/src/main/java/org/apache/parquet/variant/VariantUtil.java

```diff
@@ -187,6 +187,29 @@ class VariantUtil {
   // The size (in bytes) of a UUID.
   static final int UUID_SIZE = 16;
 
+  // header bytes
+  static final byte HDR_NULL = primitiveHeader(NULL);
```

Comment:
Nit: Omitting vowels doesn't help much. The code is less readable and it's not actually easier to type hdr than it is to type header.